os/se/ameba/rtl_se_crypto_function.h

```diff
@@ -124,3 +124,22 @@ typedef struct {
 	hal_key_type factory_slot_key_type[USABLE_FACTORY_KEY_INDEX];
 } factory_struc;
 
+/* Kernel Image Verification Structure */
+#include "amebahp_secure_boot.h"

```

Comment:
This causes build break on RTL8721CSM. Please find it on CI build test.

Reply:
Hi @sunghan-chang ,
Apologies for the inconvenience caused, added the CONFIG_AMEBASMART_TRUSTZONE.
Thank You.